### DISCUSSION TITLE
WebdriverioV9 -> isDisplayedInViewport()  removed

### DISCUSSION THREAD

**Discussion by ghost** [webdriverio/webdriverio#13471] [Technical Discussion]
Hello it seems that the function isDisplayedInViewport has been removed in V9.
Is there any function to replace this one ?
Thank you :)

**Comment by christian-bromann** [comment]
Yes, it's `await browser.isDisplayed({ withinViewport: true })`

**Reply by aliankarali** [reply]
Thanks for the solution, the type documentation are still pointing to isDisplayedInViewport().

**Reply by christian-bromann** [reply]
@aliankarali 🤔 can you point out where we still reference `isDisplayedInViewport`?

**Reply by aliankarali** [reply]
https://github.com/webdriverio/webdriverio/blob/7edf83bd122afce89f989f2c77248bf812a1a4fc/packages/webdriverio/src/commands/element/isDisplayed.ts#L14

**Reply by christian-bromann** [reply]
Ah, thanks, I will update that!
